cfme/tests/test_rest.py

```diff
@@ -848,14 +849,7 @@ def test_delete_arbitration_settings_from_detail(self, appliance, arbitration_se
         Metadata:
             test_flag: rest
         """
-        for setting in arbitration_settings:
-            del_action = getattr(setting.action.delete, method.upper())
-            del_action()
-            assert_response(appliance)
-
-            with pytest.raises(Exception, match='ActiveRecord::RecordNotFound'):
-                del_action()
-            assert_response(appliance, http_status=404)
+        delete_resources_from_detail(arbitration_settings, method)
```

Comment:
**Required**

You need to pass `not_found=True` here (and below) to actually keep the same behavior. The default of `False` means this isn't trying to delete twice anymore, and isn't matching the 'RecordNotFound' exception.